### PR TITLE
DTSSTCI-352-v2

### PR DESCRIPTION
**Pull request by gmmagruder**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/DTSSTCI-352

### Change description ###
Update existing case details page to warn users to not use non numeric characters. Also fix the banner.
Screenshot 2024-02-20 at 13 05 11

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: src/main/steps/dss-update/casefinder/content.test.ts [hunk 1]
```diff
@@ -6,12 +6,13 @@ const enContent = {
...
   line1: 'You can find this information in the email you received after submitting your tribunal forms.',
   subtitle: 'Case reference number',
-  caseNumberHint: 'This number will be 16 digits long.',
+  caseNumberHint: 'This number will be 16 digits long. It should not include spaces or any of these characters: - * (  )  &  !  /  ;',
   errorSummaryMessage: 'There is a problem',
   errors: {
     applicantCaseId: {
       required: 'Please enter a reference number',
-      notNumeric: 'Please enter a valid reference number',
+      notNumeric: 'Please enter a valid reference number. The reference number should not include spaces or any characters - * ( ) & ! /',
```
Comment (rajaram-nagarajan) [inline src/main/steps/dss-update/casefinder/content.test.ts:14]: semicolon (;) is missing from this line

File: src/main/steps/dss-update/casefinder/content.test.ts [hunk 2]
```diff
@@ -22,12 +23,13 @@ const cyContent = {
...
     'You can find this information in the email you received after submitting your tribunal forms. - welsh',
   subtitle: 'Case reference number - welsh',
-  caseNumberHint: 'This number will be 16 digits long - welsh.',
+  caseNumberHint: 'This number will be 16 digits long. It should not include spaces or any of these characters: - * (  )  &  !  /  ; - welsh.',
   errorSummaryMessage: 'There is a problem - welsh',
   errors: {
     applicantCaseId: {
       required: 'Please enter a reference number - welsh',
-      notNumeric: 'Please enter a valid reference number - welsh',
+      notNumeric: 'Please enter a valid reference number. The reference number should not include spaces or any characters - * ( ) & ! / - welsh',
```
Comment (rajaram-nagarajan) [inline src/main/steps/dss-update/casefinder/content.test.ts:31]: semicolon (;) is missing from this line

File: src/main/steps/dss-update/casefinder/content.ts [hunk 3]
```diff
@@ -9,12 +9,13 @@ export const en = () => ({
...
   line1: 'You can find this information in the email you received after submitting your tribunal forms.',
   subtitle: 'Case reference number',
-  caseNumberHint: 'This number will be 16 digits long.',
+  caseNumberHint: 'This number will be 16 digits long. It should not include spaces or any of these characters: - * (  )  &  !  /  ;',
   errorSummaryMessage: 'There is a problem',
   errors: {
     applicantCaseId: {
       required: 'Please enter a reference number',
-      notNumeric: 'Please enter a valid reference number',
+      notNumeric: 'Please enter a valid reference number. The reference number should not include spaces or any characters - * ( ) & ! /',
```
Comment (rajaram-nagarajan) [inline src/main/steps/dss-update/casefinder/content.ts:17]: semicolon (;) is missing from this line

File: src/main/steps/dss-update/casefinder/content.ts [hunk 4]
```diff
@@ -25,12 +26,13 @@ export const cy = () => ({
...
     'You can find this information in the email you received after submitting your tribunal forms. - welsh',
   subtitle: 'Case reference number - welsh',
-  caseNumberHint: 'This number will be 16 digits long - welsh.',
+  caseNumberHint: 'This number will be 16 digits long. It should not include spaces or any of these characters: - * (  )  &  !  /  ; - welsh.',
   errorSummaryMessage: 'There is a problem - welsh',
   errors: {
     applicantCaseId: {
       required: 'Please enter a reference number - welsh',
-      notNumeric: 'Please enter a valid reference number - welsh',
+      notNumeric: 'Please enter a valid reference number. The reference number should not include spaces or any characters - * ( ) & ! / - welsh',
```
Comment (rajaram-nagarajan) [inline src/main/steps/dss-update/casefinder/content.ts:34]: semicolon (;) is missing from this line

**Review by rajaram-nagarajan — CHANGES_REQUESTED** [review]

**Review by rajaram-nagarajan — APPROVED** [review]
LGTM

**Review by oleon23 — APPROVED** [review]

**Pull request merged**